linking/static/hp-socket/linked-against-hp-socket.yml

```
@@ -0,0 +1,30 @@
+rule:
+  meta:
+    name: linked against hp-socket
+    namespace: linking/static/hp-socket
+    authors:
+      - still@teamt5.org
+    scope: file
```

capa now has initial support for dynamic analysis. So rules must provide their scopes in both static (file, function, basic block, instruction, or unsupported) and dynamic (file, process, thread, call, or unsupported) analysis flavors. Please check the updated [doc/format.md](https://github.com/mandiant/capa-rules/blob/master/doc/format.md#meta-block) for more info :]

Please consider the following suggestion:
```suggestion
    scopes:
      static: file
      dynamic: file
```